examples/cloud-composer-cicd/cloudbuild.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
```

Comment:
Just FYI, I have a pull request extracting out the first two steps into its own cloud builder in the cloud builder community to avoid running it for every build.
Maybe we can incorporate it once it is merged.
https://github.com/GoogleCloudPlatform/cloud-builders-community/pull/346